src/requirements.txt

```diff
@@ -6,3 +6,4 @@ portscan==1.1
 pyfiglet==1.0.2
 speedtest-cli==2.1.3
 urllib3==2.1.0
+ansible==9.2.0
```

Comment:
Good catch!!  I had this in my develop branch - but this is a very subtle failure in the original code.